### PR TITLE
initial publish of github-actions buster version

### PR DESCRIPTION
**Pull request by arroyc**
- [X] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [X] Tests are included and/or updated for code changes.
- [X] Proper license headers are included in each file.

### CODE REVIEW

File: tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs [hunk 1]
```diff
@@ -861,6 +861,7 @@ public void SubsequentBuilds_CopyOutput_ToOutputDirectory_NestedUnderSourceDirec
 
         [Theory]
         [InlineData("github-actions")]
+        [InlineData("github-actions-buster")]
```
Comment (qianz2) [inline tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs:864]: Could you also test `github-actions-buster` image in Node and Ruby sample apps test if applicable.

File: tests/Oryx.BuildImage.Tests/Php/PhpDynamicInstallationTest.cs [hunk 2]
```diff
@@ -27,8 +27,10 @@ public PhpDynamicInstallationTest(ITestOutputHelper output) : base(output)
             {
                 var data = new TheoryData<string, string>();
                 data.Add("7.3", ImageTestHelper.WithRestrictedPermissions().GetGitHubActionsBuildImage());
+                data.Add("7.3", ImageTestHelper.WithRestrictedPermissions().GetGitHubActionsBuildImage("buster"));
```
Comment (qianz2) [inline tests/Oryx.BuildImage.Tests/Php/PhpDynamicInstallationTest.cs:30]: I think here should be like `.GetGitHubActionsBuildImage(_gitHubActionsBuster)`, use the constant created in the ImageTestHelper.

File: tests/Oryx.BuildImage.Tests/Php/PhpDynamicInstallationTest.cs [hunk 3]
```diff
@@ -27,8 +27,10 @@ public PhpDynamicInstallationTest(ITestOutputHelper output) : base(output)
...
                 var imageHelper = new ImageTestHelper();
                 data.Add("7.4", imageHelper.GetGitHubActionsBuildImage());
+                data.Add("7.4", imageHelper.GetGitHubActionsBuildImage("buster"));
```
Comment (qianz2) [inline tests/Oryx.BuildImage.Tests/Php/PhpDynamicInstallationTest.cs:33]: same here.

File: tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs (hunk 1 above)
Comment (arroyc) [reply tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs:864]: i can add one for node .. but github action doesn't have ruby .. i added tests for the new images whereever I saw a test for github-actions current image

File: build/buildBuildImages.sh [hunk 4]
```diff
@@ -158,14 +162,28 @@ function buildBuildScriptGeneratorImage() {
 }
 
 function buildGitHubActionsImage() {
+	local debianFlavor=$1
```
Comment (qianz2) [inline build/buildBuildImages.sh:165]: maybe we could extend this function for not only github-actions use. Let it pass in an image tag(could be github-actions image, jamStack image or vso images), and return the buster based images accordingly.

**Review by qianz2 — APPROVED** [review]

**Pull request merged**